docs/rules/no-attributes-during-construction.md

```diff
@@ -0,0 +1,51 @@
+# Disallow attributes during construction (no-attributes-during-construction)
+
+The `LightningElement` base class extended by LWC component classes defines several properties that, when set,
+causes its custom element to gain attributes. By
```

Comment:
Not sure we should introduce the concept of reflection here, it might cause confusion. I went with the wording used in the spec but I guess that might be confusing as well. What do you think about "renders attributes to its custom element"?

Reply:
Good point. I would replace custom element by host element: `renders attributes to its host element`